smarts/core/sensors.py

```diff
@@ -930,3 +961,36 @@ def paths_for_lane(self, lane, overflow_offset=None):
 
     def teardown(self):
         pass
+
+
+class AccelerometerSensor(Sensor):
+    def __init__(self, vehicle, sim):
+        self.linear_accelerations = deque(maxlen=3)
+        self.angular_accelerations = deque(maxlen=3)
+
+    list()
+
+    def __call__(self, linear_velocity, angular_velocity):
+        if not linear_velocity is None:
+            self.linear_accelerations.append(linear_velocity)
+        if not angular_velocity is None:
+            self.angular_accelerations.append(angular_velocity)
+
+        if len(self.linear_accelerations) < 2 or len(self.angular_accelerations):
+            return (None, None, None, None)
+
+        linear_acc = self.linear_accelerations[0] - self.linear_accelerations[1]
+        last_linear_acc = self.linear_accelerations[1] - self.linear_accelerations[2]
```

Comment:
Do we need to ensure `self.linear_accelerations` has at least 3 items?

Reply:
I thought I had but it does now.